### PR TITLE
Help text for the custom data path control

### LINKED ISSUE #301: Help text for the custom data path control.
**Branch:** `develop` (os x installers)
**Expected behavior:** Create help text for KA Lite custom data path control at KA Lite preferences dialog.

### PR DESCRIPTION
**Pull request by amodia**
@cpauya this fixes #301 

I add another window to show the help text for the KA Lite data path. 

I attached a sample screenshot of the KA Lite preferences dialog.

screen shot 2015-12-04 at 6 19 40 pm

### CODE REVIEW

**Comment by cpauya** [comment]
@amodia Let's use `UIWebView` as we discussed - https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIWebView_Class/index.html

**Comment by amodia** [comment]
@cpauya & @aronasorman I used tooltip to show the help text for the KA Lite data. This is the sample screenshot.

screen shot 2015-12-07 at 5 51 57 pm

**Comment by cpauya** [comment]
Looks good, but why do we have a lot of non-help-text-related codes here?

File: osx/post_installation.sh [hunk 1]
```diff
@@ -113,12 +113,12 @@ update_env
...
-if [ ! -d "$LAUNCH_AGENTS" ]; then
-    echo ".. Must create '$LAUNCH_AGENTS' folder..."
-    sudo mkdir -p $LAUNCH_AGENTS
+echo "$STEP/$STEPS. Remove plist in /Library/LaunchAgents folder..."
+if [ ! -f "$PLIST_SRC" ]; then
+    echo ".. Must remove '$PLIST_SRC' file..."
+    sudo rm -rf $PLIST_SRC
     if [ $? -ne 0 ]; then
-        echo ".. Abort!  Error encountered creating '$LAUNCH_AGENTS' directory."
+        echo ".. Abort!  Error encountered removing '$PLIST_SRC' file."
```
Comment (cpauya) [inline osx/post_installation.sh]: @amodia Why do you have changes to the `post_installation.sh` script of @arceduardvincent?  I'm hesitant to merge this with his feedback on these.

File: osx/post_installation.sh [hunk 2]
```diff
@@ -113,12 +113,12 @@ update_env
...
 
 ((STEP++))
-echo "$STEP/$STEPS. Create plist in ~/Library/LaunchAgents folders..."
-if [ ! -d "$LAUNCH_AGENTS" ]; then
-    echo ".. Must create '$LAUNCH_AGENTS' folder..."
-    sudo mkdir -p $LAUNCH_AGENTS
+echo "$STEP/$STEPS. Remove plist in /Library/LaunchAgents folder..."
+if [ ! -f "$PLIST_SRC" ]; then
+    echo ".. Must remove '$PLIST_SRC' file..."
+    sudo rm -rf $PLIST_SRC
```
Comment (cpauya) [inline osx/post_installation.sh]: This is a very dangerous command, let's peer-review this @amodia.

**Comment by amodia** [comment]
@cpauya I have that changes because I merge "feature/288-implement-auto-start-on-login' into feature" to this branch.

**Comment by amodia** [comment]
Merging develop branch.

**Comment by cpauya** [comment]
Ok this is better, merging.

**Pull request merged**